db/schema.sql

```diff
@@ -0,0 +1,82 @@
+DROP DATABASE  IF EXISTS  productsapi;
```

Comment:
this is a much better way to drop the database. I am going to do this in my repo. Cheers!